### DISCUSSION TITLE
emitTo multiple component

### DISCUSSION THREAD

**Discussion by omaresmael** [livewire/livewire#4600] [3. Feature ideas]
Hey guys
I had a situation where I needed to `emitTo` multiple components at once

for now, I was looping through the components and emit to them one by one, but I think it might be a better way 
**so first**: are we have this feature already implemented in livewire
**second**: if not, is it possible to do it, of the structure of livewire won't allow it 

thanks!

**Comment by joshhanley** [comment]
@omaresmael I don't think that currently exists, but PR's are welcome! I'd imagine passing an array of component names as the first parameter of `emitTo`, is that how you'd hope to use it?
